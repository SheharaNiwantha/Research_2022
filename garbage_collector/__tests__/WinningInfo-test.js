import React from "react";
import {create} from 'react-test-renderer';
import WinningInfo from "../src/screen/client_tab/winningInfoTab";

const tree = create(<WinningInfo/>);

jest.runAllTimers();

test('snapshot',()=>{
    expect(tree).toMatchSnapshot();
})