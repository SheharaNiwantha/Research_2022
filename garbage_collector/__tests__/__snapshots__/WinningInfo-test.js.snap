// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#fff",
        "height": 267,
        "justifyContent": "center",
        "width": 750,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fff",
          "height": 200,
          "justifyContent": "center",
          "width": 705,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#000",
            "fontSize": 18,
            "fontWeight": "700",
          }
        }
      />
    </View>
  </View>
</View>
`;
